frontend/src/components/buttons/DreamMallButton.vue

```diff
@@ -242,7 +242,7 @@
         y2="924.479"
         gradientUnits="userSpaceOnUse"
       >
-        <stop offset="0.0149795" stop-color="#2CA5B1" />
+        <stop offset="0.0149795" :stop-color="props.isNotification ? '#F09630' : '#2CA5B1'" />
```

Comment:
Because you do this 5 times in this file, what about:
```suggestion
        <stop offset="0.0149795" :stop-color="stopColor" />
```

Followed by:
```ts
const stopColor = props.isNotification ? '#F09630' : '#2CA5B1'
```

Reply:
Good point :+1: